File: java/app/src/main/java/net/fbvictorhugo/j/barreirasanitaria/data/dao/dummy/UsuarioDummyDAO.java
package net.fbvictorhugo.j.barreirasanitaria.data.dao.dummy;

import net.fbvictorhugo.j.barreirasanitaria.data.dao.IUsuarioDAO;
import net.fbvictorhugo.j.barreirasanitaria.data.model.Usuario;

import java.util.Date;

public class UsuarioDummyDAO implements IUsuarioDAO {

    public UsuarioDummyDAO() {

    }

    @Override
    public Usuario procurarUsuario(String login, String senha) {
        if (login != null && !login.isEmpty() && senha != null && !senha.isEmpty()) {
            Usuario usuario = new Usuario();
            usuario.setId(1);
            usuario.setLogin(login);
            usuario.setSenha(senha);
            usuario.setNome("Victor Hugo");
            usuario.setUltimoLogin(new Date());
            return usuario;
        } else {
            return null;
        }
    }
}

File: kotlin/app/src/main/java/net/fbvictorhugo/k/barreirasanitaria/data/dao/dummy/UsuarioDummyDAO.kt
package net.fbvictorhugo.k.barreirasanitaria.data.dao.dummy

import net.fbvictorhugo.k.barreirasanitaria.data.dao.IUsuarioDAO
import net.fbvictorhugo.k.barreirasanitaria.data.model.Usuario

class UsuarioDummyDAO : IUsuarioDAO {

    override fun procurarUsuario(login: String, senha: String): Usuario? {

        return if (login.isEmpty() || senha.isEmpty()) {
            val usuario = Usuario(1,"Victor Hugo")
            usuario.id = 1
            //   usuario.login = login
            //   usuario.senha = senha
            usuario.nome = "Victor Hugo"
            //  usuario.ultimoLogin = Date()
            usuario

        } else null

    }
}
